app/src/main/kotlin/compiler/intermediate_form/FunctionsUtilities.kt

```diff
@@ -13,15 +13,11 @@ import compiler.common.reference_collections.combineReferenceSets
 import compiler.common.reference_collections.referenceKeys
 import compiler.common.reference_collections.referenceSetOf
 
-object FunctionDependenciesAnalyzer {
+object FunctionsUtilities {
```

Comment:
I changed this name because as I see we want to pack here some various stuff.

Reply:
Probably you're right, I will restore the original name to make merging easier, if we would like to change it later, we can in a separate commit.